### PR TITLE
fix #40

### LINKED ISSUE #40: Add a CONTRIBUTING.md page
# What?

`./CONTRIBUTING.md` as discussed in:
* https://github.com/blog/1184-contributing-guidelines
* https://help.github.com/articles/setting-guidelines-for-repository-contributors/

# Why?

So that we have a clear path to anyone who wants to discuss rules or submit a PR with minimum confusion and higher efficiency.

# How?

This is an initial text I had in mind. Any addition/edition? Discuss on this issue.

````
## Discussion

Since this linting config is used in code bases affect various internal and external users for code that are shipped to several million people, it's important to seek consensus before changing the rules.

Opening an issue and explaining your case would be the best way to start the discussion.
If you work at Schibsted, join `#eslint-config-schibst` Slack channel.

## Publishing

```bash
# Make sure to only make do this in the package that has changed
# Update CHANGELOG
$ npm version <patch | minor | major>
$ git push --tags origin master
$ npm publish
```
````

*Then review the README.md file and add refs*

### PR DESCRIPTION
**Pull request by alexewerlof**
added CONTRIBUTING.md
moved the relevant section from README.md to this new file
see more info at #40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,16 +48,6 @@ or see [yarn installation guide](https://yarnpkg.com/en/docs/install) for more i
 Then simply run `npm t` from the root to test all packages in `packages` dir.
 
 
-## Publishing
```
Comment (SimenB) [inline README.md:51]: Link to contributing.md?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+## Consensus
```
Comment (SimenB) [inline CONTRIBUTING.md]: `h1` at the top?

**Review by SimenB — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Consensus
+
+Since this config is used in code bases that affect various internal and external repos.
+It can potentially imply changes on repos with multi-million users so we need to be mindful
```
Comment (ivarconr) [inline CONTRIBUTING.md:4]: The number of users here is over-dramatic. It affects a lot of developers in a lot of teams, but it does not directly affect the users. In many cases the code is even transpired after linting and it is mostly about making it easy to write code of high quality, with consistent syntax.

Comment (alexewerlof) [reply CONTRIBUTING.md:4]: I get the point. However I was thinking about the project I'm working on (SPiD SDK.JS) which has a couple of million users so we are very careful with every line of code. A simple change to the rule set may require us to touch many lines of code, which potentially can introduce security vulnerability to the whole schibsted (or the brands that use us like FINN, AB, Blocket, etc.)
But maybe I didn't write it in a good way? I just wanted to say: "don't cowboy around, these configs are gonna affect sensitive code". 💭 ?

**Pull request merged**